File: solutions/beecrowd/2437/2437.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[x-m y-m x-r y-r] (->> (str/split line #" ")
                                   (map #(Integer/parseInt %)))]
        (println  (+ (Math/abs (- x-r x-m))
                     (Math/abs (- y-r y-m)))))
      (recur (read-line)))))

(main)

File: solutions/beecrowd/2437/2437.lua
while true do
    local line = io.read()
    if line == nil then break end

    for x_m, y_m, x_r, y_r in string.gmatch(line, "(%w+) (%w+) (%w+) (%w+)") do
        print(
            math.abs(tonumber(x_r) - tonumber(x_m)) + math.abs(tonumber(y_r) - tonumber(y_m))
        )
    end
end
